docs/sdk/wallets/minting-methods/mint-via-web-authn.md

```diff
@@ -0,0 +1,22 @@
+# Mint via WebAuthn
+
+You can mint a PKP by presenting a valid WebAuthn credential generated by your browser to the Lit Relay server. 
+
+We have a frontend that helps with this process at https://pkp-walletconnect.vercel.app/.
```

Comment:
this demo has been deprecated

Reply:
demo here: https://lit-pkp-auth-demo.vercel.app/

Reply:
Updated @a1ttech!
Thanks :)